docs/standard/native-interop/interop-exceptions.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "Platform Neutral Interop Exceptions"
+ms.date: "01/16/2020"
+ms.technology: dotnet-standard
+helpviewer_keywords:
+  - "unmanaged code, exceptions"
+  - "exceptions, unmanaged code"
+  - "interop, exceptions"
+  - "exceptions, interop"
+  - "stack, jump"
```

Comment:
"stack jump" does not seem like a combination of keywords that somebody looking for this doc is going to use. Bing or Google does not show anything even remotely related when searching for "stack jump .NET".